src/ui/layout/responsive/building-adaptive-apps.md

```diff
@@ -397,6 +403,16 @@ return Padding(
 );
 ```
 
+Use `Theme.of(context).platform` for theming and
+design choices, like what kind of switches to show
+and general Cupertino/Material adaptions.
+It also helps an app work well on mobile web.
```

Comment:
From reading, it's not clear to me how this helps an app work well on mobile web. Consider elaborating or dropping the sentence.